R/internal.R

```diff
@@ -0,0 +1,9 @@
+.dashDangerouslySetInnerHtml_js_metadata <- function() {
+deps_metadata <- list(`dash_dangerously_set_inner_html` = structure(list(name = "dash_dangerously_set_inner_html",
+version = "0.0.3", src = list(href = NULL,
+file = "deps"), meta = NULL,
+script = 'bundle.js',
```

Comment:
At some point, I'd actually suggest that this package rename its bundle to something more descriptive, as we have with other projects, but it's not essential for this PR.